src/lib/ui/Workspace.svelte

```diff
@@ -9,6 +9,8 @@
 	export let friends: Member[];
 	export let communities: Community[];
 	let selected_community = communities[0] || null;
+	$: selected_community_members = selected_community.members;
```

Comment:
This will error if `selected_community` is null, which can happen if there are no loaded communities. (e.g. a new account). Copying the pattern below:

```suggestion
	$: selected_community_members = selected_community?.members || null;
```